lib/QubitCsvTransformFactory.class.php

```diff
@@ -174,6 +178,8 @@ public function make()
                         if ($levelOfDescriptionAvailable) {
                             if (!empty($self->preserveOrder)) {
                                 $sortorder = $self->getStatus('rows');
+                            } elseif (!empty($self->sortOrderLogic)) {
+                                $sortorder = call_user_func_array($self->sortOrderLogic, [$self]);
```

Comment:
I think this will throw an fatal error if `$self->sortOrderLogic` isn't callable in the current scope, which may be the desired behaviour.  It could be made a bit more user friendly by testing `is_callable($self->sortOrderLogic)` and throwing a more user friendly error message, but this might be overkill if we are expecting the transformation script to be written by an experienced PHP developer.